.github/workflows/pr.yml

```diff
@@ -75,12 +80,21 @@ jobs:
         with:
           version: "2021.1.2"
           install-go: false
-      - name: Check manifests
+      - name: Check kcp manifests
+        run: |
+          CONTROL_PLANE=kcp make generate manifests
+          if [[ ! -z $(git status ':(exclude)bin/kustomize' ':(exclude)bin/controller-gen' -s) ]]
+          then
+            echo "generated kcp sources are not up to date:"
+            git --no-pager diff ':(exclude)bin/kustomize' ':(exclude)bin/controller-gen'
+            exit 1
+          fi
+      - name: Check kubernetes manifests
         run: |
-          make generate manifests
+          CONTROL_PLANE=kubernetes make generate manifests
```

Comment:
You don't really need to specify the control plane here although if you feel strong about keeping it here I'm fine with it.

Reply:
I'd like to keep it. I think it makes it easier to read when contrasting with the kcp check